src/pages/workspace/WorkspaceInitialPage.tsx

```diff
@@ -285,7 +290,8 @@ function WorkspaceInitialPage({policyDraft, policy: policyProp, policyMembers, r
                                     onPress={item.action}
                                     brickRoadIndicator={item.brickRoadIndicator}
                                     wrapperStyle={styles.sectionMenuItem}
-                                    focused={!!(item.routeName && activeRoute?.startsWith(item.routeName))}
+                                    highlighted={enabledItem?.name === item.name}
+                                    focused={!!(item.translationKey && activeRoute?.startsWith(item.translationKey))}
```

Comment:
`activeRoute?.startsWith(item.translationKey)` is this correct? does not feel right to me

Reply:
Yep, seems like a leftover. Will change it.